test/regression.uts

```diff
@@ -762,9 +762,10 @@ assert ASN1_Object(1) != ASN1_Object(2)
 ASN1_Object(2).show()
 
 = ASN1 - RandASN1Object
+~ random_weird_py3
 a = RandASN1Object()
 random.seed(0x2807)
-assert raw(a) in [b'A\x02\x07q', b'C\x02\xfe\x92', b'\x1e\x023V']
+assert raw(a) in [b'A\x02\x07q', b'C\x02\xfe\x92', b'\x1e\x023V', b'\x02\x02\xfe\x92']
```

Comment:
Is the list still needed with the new `random_weird_py3` tag ?

Reply:
Not sure, I'll check.

Reply:
I've pushed a fix. If it works, I'll fixup the commit and push again.

Reply:
So it seems it was not needed, nice spot. Tests are running, OK to merge for me.